### ISSUE TITLE
Feature Request: A http-get action for the macro

### ISSUE THREAD

**Issue by drakejest** [WarmUpTill/SceneSwitcher#514]
I have been working with PTZ ipcamera as a video source in OBS. I could not find a plugin wherein if i transition to another scene in OBS my camera would move to a certain position, when i say move i mean it will physically move to look at a certain position. There are other plugins that specifically controls IP cameras but they are manual control or when they are automated they lack the flexibility and versatility of the triggers that your plugin offers. One way of controlling the PTZ cameras is via an http-get request and your plugin has currently no option for a http-get request in the macros.

My current solution to this is using SceneSwitcher with AutoHotkey. For example, i have created a macro that when the current scene is particular scene, the macro would execute a hotkey of 'f13'. This f13 would trigger  AutoHotkey to do a http-get request with url: `http://192.168.1.108/cgi-bin/ptz.cgi?action=start&channel=0&code=Up&arg1=0&arg2=1&arg3=0`. This will then move the PTZ ipcamera to a certain position. I would like to eliminate the use of another program, as it can be a point of failure.

To summarize, in the macro section please add an action that will do an http-request, the response can just be ignored. Here is an image what i think it should look like 

![image](https://user-images.githubusercontent.com/62618765/187983349-f5c7238e-600f-4a80-bf0d-dbb807216cc4.png)

Adding this feature does not only benefit PTZ ipcameras but also other network IoT devices such as LED strip controller that can be controlled via http-request.

Thank you ! :) hope you have a great day.

**Comment by nacalai** [comment]
He literally just added http action :) It's already there

**Comment by WarmUpTill** [comment]
Thanks for the suggestion - as already mentioned, this was implemented recently in #492  :)
